python/kserve/kserve/model_server.py

```diff
@@ -127,9 +127,11 @@ def start(self, models: Union[List[Model], Dict[str, Deployment]]) -> None:
         if isinstance(models, list):
             for model in models:
                 if isinstance(model, Model):
-                    self.register_model(model)
-                    # pass whether to log request latency into the model
-                    model.enable_latency_logging = self.enable_latency_logging
+                    print(model.name, "ready status is", model.ready)
+                    if model.ready:
```

Comment:
I think we can raise the exception if any model is not ready during start up

Reply:
You mean that an exception is triggered as long as one model is not ready? Because the initial idea was that an exception would be triggered only if none of the models is ready.